javaslang/src-gen/main/java/javaslang/control/Applicative.java

```diff
@@ -0,0 +1,439 @@
+/*     / \____  _    _  ____   ______  / \ ____  __    _______
+ *    /  /    \/ \  / \/    \ /  /\__\/  //    \/  \  //  /\__\   JΛVΛSLΛNG
+ *  _/  /  /\  \  \/  /  /\  \\__\\  \  //  /\  \ /\\/ \ /__\ \   Copyright 2014-2016 Javaslang, http://javaslang.io
+ * /___/\_/  \_/\____/\_/  \_/\__\/__/\__\_/  \_//  \__/\_____/   Licensed under the Apache License, Version 2.0
+ */
+package javaslang.control;
+
+/*-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-*\
+   G E N E R A T O R   C R A F T E D
+\*-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-*/
+
+import java.util.function.BiFunction;
+import java.util.function.Function;
+import javaslang.collection.List;
+import javaslang.concurrent.Future;
+
+import javaslang.Function1;
+import javaslang.Function2;
+import javaslang.Function3;
+import javaslang.Function4;
+import javaslang.Function5;
+import javaslang.Function6;
+import javaslang.Function7;
+import javaslang.Function8;
+import javaslang.collection.List;
+import javaslang.control.Option;
+import javaslang.control.Try;
+import javaslang.concurrent.Future;
+import java.util.function.BiFunction;
```

Comment:
\[Checkstyle\] ERROR: Duplicate import to line 12 \- java\.util\.function\.BiFunction\.